tg_bot.py

```diff
@@ -34,9 +34,17 @@ def start(self, update, context):
                     pass
                 url = 'https://twitter.com/' + \
                     username + '/status/' + str(line)
-                keyboard = [[InlineKeyboardButton(
-                    "Retweet", callback_data=int(line))]]
-                reply_markup = InlineKeyboardMarkup(keyboard)
+                tg_id = int(update.message.from_user.id)
+                
+                if tg_id in config.tg_admin_id:
```

Comment:
Does it iterate through the list to check if all the admin ids are checked?

I'm wondering if any() is of use in here?

Reply:
Also, need to check if two users initiate the bot at the same time - does it prevents us from a normal use case or not?

Reply:
> Also, need to check if two users initiate the bot at the same time - does it prevents us from a normal use case or not?

I don't think it would be a problem as bots are used at the same time, but if you want to check, we can :)

Reply:
It's better to have it tested than leave it untested

Reply:
> Does it iterate through the list to check if all the admin ids are checked? I'm wondering if any() is of use in here?

The if statement just checks whether the user id is there in the list of admin id, if yes it will create the retweet button.

any() will just check whether the list is iterable or not, why would we need to do that?